packages/garbo/src/familiar/barfFamiliar.ts

```diff
@@ -294,6 +301,18 @@ function getSpecialFamiliarLimit({
     case $familiar`Crimbo Shrub`:
       return Math.ceil(estimatedGarboTurns() / 100);
 
+    // If we're going to ascend, Chest Mimic can't be leveled before the next embezzler chain
+    // If we're not going to ascend, Chest Mimic shouldn't be charged beyond what it can spit out tomorrow
+    case $familiar`Chest Mimic`:
+      return globalOptions.ascend
+        ? 0
+        : !shouldChargeMimic()
+        ? 0
+        : $familiar`Chest Mimic`.experience < 550
+        ? ((get("valueOfAdventure") * EMBEZZLER_MULTIPLIER()) / 50) *
+          getModifier("Familiar Experience")
+        : 0;
```

Comment:
This should be one big condition in the ternary, or you can use curly braces to do multiple lines of logic. but not a giant ternary